src/sidebar/store/modules/test/filters-test.js

```diff
@@ -45,6 +45,15 @@ describe('sidebar/store/modules/filters', () => {
         assert.isFalse(filterState.focusActive);
         assert.isUndefined(filterState.focusFilters.user);
       });
+
+      it('un-sets and deactivates the user focus filter when null user value is provided', () => {
```

Comment:
To check my understanding - setting the user to `null` means "the focus mode should not be available" whereas setting it to `{ username: undefined, displayName: undefined }` is a way of toggling with it is active or not, but it remains available for the user to toggle?

It would be good to try and find a more intuitive API for the latter case if so.